src/scripts/main.js

```diff
@@ -1,5 +1,43 @@
 'use strict';
 
 document.addEventListener('click', (e) => {
-  // write code here
+  const spider = document.querySelector('.spider');
+  const wall = document.querySelector('.wall');
+  const target = e.target.closest('.wall');
+
+  if (!target) {
+    return;
+  }
+
+  spider.style.left = coords(
+    e.clientX,
+    wall.offsetLeft,
+    spider.offsetWidth,
+    wall.clientWidth,
+    wall.clientLeft,
```

Comment:
The 'coords' function is called with five arguments here, but it is defined to accept only five parameters. The sixth parameter 'wall.clientLeft' seems unnecessary and should be removed from the function call.